src/common.ts

```diff
@@ -11,3 +11,21 @@
 export type FilePath = string;
 /** Type alias to indicate the path of a directory. */
 export type DirPath = string;
+
+/**
+ * Use this with `.catch()` on promises where the error is already
+ * handled elsewhere, but where you would like to debug log the
+ * condition.
+ */
+export function catchFloatingPromise(err: Error) {
```

Comment:
Maybe this can be named to be more descriptive of it's behaviour rather than the characteristics of where it's used -> `passthroughErrorLogger` ?

Reply:
Fixed, thanks.